Dockerfiles/syslog-ng-incubator-dev/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM balabit/syslog-ng-dev:latest
+MAINTAINER Andras Mitzki <andras.mitzki@balabit.com>
+
+ENV PKG_CONFIG_PATH=$PKG_CONFIG_PATH:/usr/lib/pkgconfig/
+
+RUN apt-get update -qq && apt-get install -y \
+    gperf \
+    libevent-dev \
+    liblua5.2-dev \
+    libmongo-client-dev \
+    libperl-dev \
+    librdkafka-dev \
+    libtokyocabinet-dev \
+    libzmq3-dev \
+    syslog-ng-dev
+
+RUN git clone https://github.com/talien/grok.git /root/grok && \
+    cd /root/grok && \
+    autoreconf -i && \
+    ./configure && \
+    make && \
+    make install
+
+ADD pip-dependencies.txt .
```

Comment:
I'd call this simply `requirements.txt`.  We don't have a lot of files and this name is the convention.